purchase_sale_inter_company/models/sale_order.py

```diff
@@ -16,12 +17,42 @@ class SaleOrder(models.Model):
         copy=False,
     )
 
+    def assert_intercompany_prices_equal(self):
+        """Check if the prices of both orders are the same"""
+        unequal_line = []
+        for so in self:
+            for line in so.sudo().order_line:
+                if not so.currency_id.compare_amounts(
+                    line.price_unit, line.auto_purchase_line_id.price_unit):
+                    continue
```

Comment:
purchase_sale_inter_company/models/sale_order.py:26:5: E125 continuation line with same indent as next logical line

Reply:
moved

Reply:
The error is now 
`purchase_sale_inter_company/models/sale_order.py:28:24: E126 continuation line over-indented for hanging indent`
Looks like you have three spaces there instead of four.